keyboards/dztech/dz96/readme.md

```diff
@@ -0,0 +1,15 @@
+# DZ96
+
+![DZ96](https://kbdfans.com/products/tofu96-mechanical-keyboard-diy-kit)
```

Comment:
This needs to be an actual image (preferably hosted on imgur).

Reply:
added one.
the only image that I could find is the tofu96 kit photo.